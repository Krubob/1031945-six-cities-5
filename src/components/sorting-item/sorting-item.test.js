import React from "react";
import renderer from "react-test-renderer";
import SortingItem from "./sorting-item";

describe(`SortingItem snapshot`, () => {
  it(`SortingItem should render correctly when type is equal activeSorting`, () => {
    const tree = renderer
    .create(
        <SortingItem
          type={`Popular`}
          activeSorting={`Popular`}
          onSortingClick={()=>{}}
        />
    )
    .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it(`SortingItem should render correctly when type not equal activeSorting`, () => {
    const tree = renderer
    .create(
        <SortingItem
          type={`Top rated first`}
          activeSorting={`Popular`}
          onSortingClick={()=>{}}
        />
    )
    .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
