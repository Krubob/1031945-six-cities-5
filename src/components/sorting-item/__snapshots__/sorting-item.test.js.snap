// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SortingItem snapshot SortingItem should render correctly when type is equal activeSorting 1`] = `
<li
  className="places__option places__option--active"
  onClick={[Function]}
  tabIndex="0"
>
  Popular
</li>
`;

exports[`SortingItem snapshot SortingItem should render correctly when type not equal activeSorting 1`] = `
<li
  className="places__option "
  onClick={[Function]}
  tabIndex="0"
>
  Top rated first
</li>
`;
